src/SonarScanner.MSBuild.PostProcessor/PostProcessor.cs

```diff
@@ -222,6 +222,7 @@ private void ProcessCoverageReport(AnalysisConfig config, IBuildSettings setting
         }
         else if (settings.BuildEnvironment is BuildEnvironment.LegacyTeamBuild && !BuildSettings.SkipLegacyCodeCoverageProcessing)
         {
+            runtime.Logger.AddTelemetryMessage("scanner.legacyTFS", "Called");
```

Comment:
```suggestion
            runtime.Logger.AddTelemetryMessage("dotnetenterprise.s4net.legacyTFS", "Called");
```

Reply:
Fixed in (https://github.com/SonarSource/sonar-scanner-msbuild/pull/2775/